ruby/app/models/Truv.rb

```
@@ -33,6 +34,43 @@ def self.createUserBridgeToken(user_id)
+        "bank_name" => "Truv Bank"
+      }
+
+      if product_type == "pll"
+        bodyObj["employers"][0]["account"]["deposit_type"] = "amount"
+        bodyObj["employers"][0]["account"]["deposit_value"] = "100"
+      end
+    end
+
+    body = bodyObj.to_json
+    return sendRequest("orders/", body, "POST")
+  end
```

RuboCop (1.75.5)

[convention] Assignment Branch Condition size for `createOrder` is too high. [<7, 22, 4> 23.43/23]

(Metrics/AbcSize)